stable/osg-hosted-ce/10000

```diff
@@ -0,0 +1 @@
+sudo pmset hibernatemode 0 standby 0
```

Comment:
This is OS X specific right?

Reply:
Yes I think so, I think this file was automatically generated I'll delete it